packages/spaces/commands/vpn/config.js

```diff
@@ -65,5 +65,6 @@ You will use the information provided by this command to establish a Private Spa
     {name: 'space', char: 's', hasValue: true, description: 'space to get VPN config from'},
     {name: 'json', description: 'output in json format'}
   ],
-  run: cli.command(co.wrap(run))
+  run: cli.command(co.wrap(run)),
+  displayVPNConfigInfo: displayVPNConfigInfo
```

Comment:
WDYT about calling this `render` in the same style that `info.js` does? It might require some refactoring.

Reply:
@alindeman I considered that, but wasn't sure it was *entirely* necessary since this function wouldn't need the same conditionals as `info.js`.

Reply:
Actually, I guess I can add in the same conditionals.

Reply:
Gotcha, yah, I see that `wait` doesn't have a `json` flag.